main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -2079,7 +2079,7 @@ protected void fillViewHolder(final View rowView, final WaypointViewHolder holde
                 noteView.setOnClickListener(new DecryptTextClickListener(noteView));
                 noteView.setVisibility(View.VISIBLE);
                 if (TextUtils.containsHtml(wpt.getNote())) {
-                    noteView.setText(HtmlCompat.fromHtml(wpt.getNote(), HtmlCompat.FROM_HTML_MODE_LEGACY, new SmileyImage(cache.getGeocode(), noteView), new UnknownTagsHandler()), TextView.BufferType.SPANNABLE);
+                    noteView.setText(HtmlCompat.fromHtml(wpt.getNote(), HtmlCompat.FROM_HTML_MODE_LEGACY, new SmileyImage(cache.getShortGeocode(), noteView), new UnknownTagsHandler()), TextView.BufferType.SPANNABLE);
```

Comment:
Not sure whether the change for the SmileyImage is required or helpful at all. Same for ImagesActivity.

Reply:
Are you sure that all code places where you changed `getGeocode` to `getShortGeocode` really don't need the correct geocode? Otherwise, this might break existing functionality which wouldn't be worth it...

Reply:
I understand the risks involved and no, I am not sure, which is the reason I have already started to backout some of those changes. And I fully agree, it wouldn't be worth it.